packages/manager/modules/pci/src/projects/project/notebooks/notebooks.module.js

```diff
@@ -34,6 +34,7 @@ angular
   .config(routing)
   .component('ovhManagerPciProjectNotebooks', component)
   .service('NotebookService', service)
-  .run(/* @ngTranslationsInject:json ./translations */);
+  .run(/* @ngTranslationsInject:json ./translations */)
+  .run(/* @ngTranslationsInject:json ../training/jobs/translations */);
```

Comment:
**issue:** A module should bring its own translation. Please avoid accessing to a parent module to get access to some translations.

Reply:
This was was made on purpose, it was a design error to have notebooks in a different module than training and I would like to get them back into the same module in the end.
Main issue I have right now is that the Training and Notebook product is very similar and right now due to different translation files being used we have non consistency across both products (making it harder for customers to understand the product)

Reply:
Hi @christophe-rannou , We think its better to copy these translations than to access like this. We want to make every component as atomic as possible.
We can copy the relevant translations with different keys.

Reply:
fixed